bootstrap.sh

```diff
@@ -179,15 +179,13 @@ function install_zookeeper() {
   zk="zookeeper-$version"
   wget "http://apache.org/dist/zookeeper/$zk/$zk.tar.gz"
   tar -xzf "$zk.tar.gz"
+  ant -f "$zk/build.xml" package
+  ant -f "$zk/src/contrib/fatjar/build.xml" jar
   mkdir -p lib
-  cp "$zk/contrib/fatjar/$zk-fatjar.jar" lib
```

Comment:
since the default version is still 3.4.13, perhaps you need to keep the workaround, and make it conditional on the version.

Reply:
The `ant ... jar` command above appears to work for 3.4.13 as well. The fatjar is copied into the same location a few lines below.

Reply:
:+1: